submissions/yaripey/frogger/app.js

```diff
@@ -0,0 +1,124 @@
+// Tile size
+const tile = {
+    width: 101,
+    height: 85
+}
+
+// Enemies our player must avoid
+const Enemy = function (x, y) {
+    // Variables applied to each of our instances go here,
+    // we've provided one for you to get started
+    this.x = x;
+    this.y = y;
+    this.changeSpeed();
+    // The image/sprite for our enemies, this uses
+    // a helper we've provided to easily load images
+    this.sprite = 'images/enemy-bug.png';
+};
+
+Enemy.prototype.changeSpeed = function () {
+    this.speed = Math.random() * (400 - 100) + 100;
```

Comment:
magic numbers...
All numbers like block dimensions, initial locations should be defined as constants.